kafka-minion/values.yaml

```diff
@@ -119,3 +119,15 @@ serviceMonitor:
   # different matchLabels configuration, to make sure it matches this ServiceMonitor.
   # If this is defined, the release label of the service monitor will match kube-eagle's one
   # additionalLabels:
+
+# Create a service account by default
```

Comment:
By default it does **not** create a service account now. The comment is missleading.

Reply:
fixed